From 23bf2e48be4306581a14a31d447ebe47fc88db39 Mon Sep 17 00:00:00 2001
From: Marek Majtyka <marekx.majtyka@intel.com>
Date: Fri, 10 Nov 2017 08:39:53 -0600
Subject: [PATCH 259/278] drivers/edac: Changed default EDAC CE log settings

Modified all axxia EDAC drivers default value of log_ce to 0.
This disables console EDAC CE errors notification.

Signed-off-by: Marek Majtyka <marekx.majtyka@intel.com>
---
 drivers/edac/axxia_edac-cmc_56xx.c    | 2 ++
 drivers/edac/axxia_edac-l2_cpu_56xx.c | 1 +
 drivers/edac/axxia_edac-l3_56xx.c     | 2 ++
 drivers/edac/axxia_edac-mc_56xx.c     | 2 ++
 4 files changed, 7 insertions(+)

diff --git a/drivers/edac/axxia_edac-cmc_56xx.c b/drivers/edac/axxia_edac-cmc_56xx.c
index 11c650af..bdc76b1 100644
--- a/drivers/edac/axxia_edac-cmc_56xx.c
+++ b/drivers/edac/axxia_edac-cmc_56xx.c
@@ -1236,6 +1236,8 @@ static int initialize(struct intel_edac_dev_info *dev_info)
 		return ERR_STAGE_1;
 	}
 
+	dev_info->edac_dev->log_ce = 0;
+
 	instance = &dev_info->edac_dev->instances[0];
 
 	/* It just gives more descriptive name. */
diff --git a/drivers/edac/axxia_edac-l2_cpu_56xx.c b/drivers/edac/axxia_edac-l2_cpu_56xx.c
index 3248384..ecab59c 100644
--- a/drivers/edac/axxia_edac-l2_cpu_56xx.c
+++ b/drivers/edac/axxia_edac-l2_cpu_56xx.c
@@ -227,6 +227,7 @@ static void intel_add_edac_devices(struct platform_device *pdev,
 		goto err1;
 	}
 
+	dev_info->edac_dev->log_ce = 0;
 	dev_info->edac_dev->pvt_info = dev_info;
 	dev_info->edac_dev->dev = &dev_info->pdev->dev;
 	dev_info->edac_dev->ctl_name = dev_info->ctl_name;
diff --git a/drivers/edac/axxia_edac-l3_56xx.c b/drivers/edac/axxia_edac-l3_56xx.c
index af4ff16..eb25eed 100644
--- a/drivers/edac/axxia_edac-l3_56xx.c
+++ b/drivers/edac/axxia_edac-l3_56xx.c
@@ -456,6 +456,8 @@ static int intel_edac_l3_probe(struct platform_device *pdev)
 		goto err1;
 	}
 
+	dev_info->edac_dev->log_ce = 0;
+
 	r = platform_get_resource(pdev, IORESOURCE_IRQ, 0);
 	if (!r)
 		return -EINVAL;
diff --git a/drivers/edac/axxia_edac-mc_56xx.c b/drivers/edac/axxia_edac-mc_56xx.c
index fcc53bd..c9429ee 100644
--- a/drivers/edac/axxia_edac-mc_56xx.c
+++ b/drivers/edac/axxia_edac-mc_56xx.c
@@ -1341,6 +1341,8 @@ static int intel_edac_mc_probe(struct platform_device *pdev)
 		goto err_noctlinfo;
 	}
 
+	dev_info->edac_dev->log_ce = 0;
+
 	instance = &dev_info->edac_dev->instances[0];
 
 	/* It just gives more descriptive name. */
-- 
1.8.5.2.233.g932f7e4

